yarn.lock

```diff
@@ -6906,6 +6906,13 @@ semver@^7.1.1, semver@^7.1.3, semver@^7.3.2, semver@^7.3.4, semver@^7.3.5:
   dependencies:
     lru-cache "^6.0.0"
 
+semver@^7.3.7:
```

Comment:
this is a little odd, all of these should have had the same version here.

have you run a `yarn` in the root directory.

Reply:
Yes, this is the yarn.lock after running in root. The only changes I have made is the cdk version to be able to use FuntionUrl.